src/UbiConstants.h

```diff
@@ -30,7 +30,7 @@ Inc
 #include "stdint.h"
 
 const char UBIDOTS_INDUSTRIAL_IP[] = "169.55.61.243";
-const char *const USER_AGENT = "UbidotsESP8266/3.0.1";
+const char *const USER_AGENT = "UbidotsESP8266/3.0.3";
```

Comment:
I think that this change is minor so, I suggest that the version be 3.1.0

Reply:
Appropriate header files for BearSSL are indirectly included by [ESP8266WiFi.h](https://github.com/esp8266/Arduino/blob/612e7ffd7fb398fc2cdcf8bd1d177b75d997d46c/libraries/ESP8266WiFi/src/ESP8266WiFi.h) which is included by [src/UbiProtocol.h](https://github.com/ubidots/ubidots-esp8266/blob/d5daa3d1d4ccbf2fd1f895d809a8be7bff546334/src/UbiProtocol.h).